src/plugins/seek_time/seek_time.js

```diff
@@ -25,7 +25,7 @@ export default class SeekTime extends UICorePlugin {
   }
   get mediaControl() { return this.core.mediaControl }
   get mediaControlContainer() { return this.mediaControl.container }
-  get durationShown() { return this.mediaControlContainer.getPlaybackType() !== Playback.LIVE || this.mediaControlContainer.isDvrInUse() }
+  get durationShown() { return this.mediaControlContainer.getPlaybackType() === Playback.LIVE && this.mediaControlContainer.isDvrEnabled() }
```

Comment:
When it's vod the duration and current time is shown in the media control
![image](https://cloud.githubusercontent.com/assets/3259993/11814740/66f0f290-a340-11e5-8734-c4ab8d191419.png)